client-app/src/examples/portfolio/detail/OHLCChartModel.js

```diff
@@ -13,7 +13,8 @@ export class OHLCChartModel {
     constructor() {
         this.addReaction({
             track: () => this.symbol,
-            run: () => this.loadAsync()
+            run: () => this.loadAsync(),
+            fireImmediately: true
```

Comment:
why is this fireImmediately needed?

Reply:
It isn't needed. I was trying to fix the somewhat flaky loading.

As you keep saying, I should look over the diff before opening a PR.